MouseAimFlight/MouseAimVesselModule.cs

```diff
@@ -275,10 +277,12 @@ void FlyToPosition(FlightCtrlState s, Vector3 targetPosition)
             Steer steer = pilot.Simulate(behavior.pitchError, behavior.rollError, behavior.yawError, localAngVel, terrainAltitude, TimeWarp.fixedDeltaTime, dynPressure, velocity);
 
             //Piloting
-            s.pitch = Mathf.Clamp(steer.pitch, -1, 1);
+            if (s.pitch == s.pitchTrim)
```

Comment:
Doesn't this disable the functionality set by "pitchYawOverrideMouseAim"?
It is in place so that you can lock pitch and yaw to avoid crashing your aircraft by accident.

Reply:
I think that since now camera will follow the target, the player now will always know where the target is, thus not needing to reset target position; instead what I did was to changed that portion of code to what was like the control scheme like in war thunder: when pitch/yaw trim is changed manually, FlyToPosition's targetPosition will set to where the vessel is pointing at rather than resetting mouse direction to vessel direction. And then, player's control will completely override the autopilot's behavior rather than roll only. 
By doing so I feel the control seems more comfortable for me. Of course if you don't like this change I can revert this back.

Reply:
[https://www.youtube.com/watch?v=DjmX45faTq0](url)
This is a video shows what will happen now if I press W/A/S/D...